README.md

```diff
@@ -19,80 +19,11 @@
 
 ### FUNCTIONALITY
 
-- ~~TODO: Complete a turn~~
-- ~~TODO: VALIDATION, do not allow user to end turn without rolling dice~~
-- ~~TODO: add end game function to stop execution?~~
-- TODO: UI implementation for frontend
-  - material-ui to make things look nice
-  - display properties
-  - move pieces
-- TODO: random array of movement emojis
-  - read: emoji manifesto
-- TODO: enforce global house limits (32), hotel limits (12)
-- TODO: game log
-- TODO: continue game
-- TODO: remaining
-  - ~~startTurn~~
-  - ~~PropertyService~~
-  - ~~MoveService?~~
-    - ~~gameState must have extra configuration~~
-  - ~~PropertyLookupService? (retrieves position / tile meta to determine auction/buy/no-op/rent)~~
-  - WealthService
-    - liquidity
-    - bankruptcy
-  - CardService
-    - USE_GET_OUT_OF_JAIL_CARD
-  - PropertyManagementService
-    - service to manage property purchasing/global maximums
-    - CONSTRUCT_HOUSE,
-    - DECONSTRUCT_HOUSE,
-    - CONSTRUCT_HOTEL,
-    - DECONSTRUCT_HOTEL,
-    - MORTGAGE_PROPERTY,
-      - MortgageService?
-    - AUCTION
-    - BUY
-    - RENT value
-      - PAY_RENT (determines rent / monopoly)
-  - BONUS:
-    - Players
-- ~~TODO: Pass in options to set fine/go parameters/max houses~~
+- TODO
```

Comment:
? What's this ?

Reply:
Moving all the TODOs into the GitHub Issues. Left one or two TODOs for sections I do want to keep and finish in the README.md proper.